homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -534,10 +544,35 @@ async def put(self, request, username, entity_number):
             )
 
         if service is not None:
+            state_will_change = parsed[STATE_ON] != (entity.state != STATE_OFF)
+
             hass.async_create_task(
                 hass.services.async_call(domain, service, data, blocking=True)
             )
 
+            if state_will_change:
+                # Wait for the state to change.
+                asyncevent = asyncio.Event()
+
+                @core.callback
+                def async_event_changed(event):
```

Comment:
```suggestion
                def async_event_changed(_):
```

We don't care about this argument so we can use `_`

Reply:
Suggested change here https://github.com/bdraco/home-assistant/commit/1e4f43e414fd3ab8d5402771d56f8fdacf8112e4